mkt/extensions/validation.py

```diff
@@ -87,6 +92,8 @@ def validate(self):
         self.validate_description(self.data)
         self.validate_version(self.data)
         self.validate_author(self.data)
+        self.validate_icons(self.data)
+        # self.validate_icon_files(self.data)
```

Comment:
I wasn't 100% clear on the engineering path forward, so I started stubbing things out here.